timesketch/api/v1/resources/sketch.py

```diff
@@ -370,6 +358,29 @@ def get(self, sketch_id):
                 mapping_dict['type'] = value_dict.get('type', 'n/a')
                 mappings.append(mapping_dict)
 
+        if sketch_indices:
+            # Stats for index. Num docs per shard and size on disk.
+            for timeline in sketch.active_timelines:
+                if indices_metadata[timeline.searchindex.index_name].get('is_legacy', False):
+                    doc_count, _ = self.datastore.count(
+                        indices=timeline.searchindex.index_name)
+                    stats_per_timeline[timeline.id] = {
+                        'count': doc_count
+                    }
+                else:
+                    # TODO: Consider using an aggregation here instead?
+                    query_string = f'__ts_timeline_id:{timeline.id}'
```

Comment:
better to use filtering (raw query dsl) or term instead of a query string

Reply:
done